cssom-view-1/Overview.bs

```diff
@@ -1011,7 +1011,7 @@ Note: Some non-conforming implementations are known to return 32 instead of 24.
 partial interface Document {
   Element? elementFromPoint(double x, double y);
   sequence&lt;Element> elementsFromPoint(double x, double y);
-  CaretPosition? caretPositionFromPoint(double x, double y);
+  CaretPosition? caretPositionFromPoint(double x, double y, ShadowRoot... shadowRoots);
```

Comment:
Are we also working on adding WPT coverage for the shadow tree cases?

Reply:
Yes. I am working on adding WPT coverage.

Reply:
If you a PR open, could you add a link to it in this PR's description? If not, you can mention there that WPT coverage will be added subsequently.